dev/Repeater/APITests/FlowLayoutTests.cs

```diff
@@ -1260,6 +1259,43 @@ public void ValidateUniformGridWithNoItems()
             });
         }
 
+        [TestMethod]
+        public void ValidateUniformGridWithItemsRelyingOnAspectRatio()
+        {
+            ItemsRepeater repeater = null;
+            ScrollViewer scroller = null;
+            RunOnUIThread.Execute(() =>
+            {
+                repeater = new ItemsRepeater() {
+                    ItemsSource = Enumerable.Range(0, 6), // no items 
+                    Layout = new UniformGridLayout() {
+                        Orientation = Orientation.Vertical,
+                        MaximumRowsOrColumns = 1,
+                        MinItemWidth = 100
+                    },
+                    MaxWidth = 300,
+                    ItemTemplate = GetDataTemplate("<repeaterCommon:AspectRatioRespectingControl/>")
+                };
+
+                scroller = new ScrollViewer() {
+                    Content = repeater
+                };
+
+                Content = scroller;
+                repeater.UpdateLayout();
+                Content.UpdateLayout();
```

Comment:
should not be required, calling updatelayout on any element runs an entire layout.